acceptance/tests/pxp_agent_version.rb

```diff
@@ -4,7 +4,8 @@
 
 agents.each do |agent|
   step 'cd into pxp-agent bin folder and check the version' do
-    on(agent, "cd #{pxp_agent_dir(agent)} && ./pxp-agent --version") do |result|
+    # Vendored Ruby needs to be on path. The "privatebindir" property of beaker host can be used for this
+    on(agent, "export PATH=\"#{agent['privatebindir']}:$PATH\" && #{pxp_agent_dir(agent)}/pxp-agent --version") do |result|
```

Comment:
well the export only persists for this single ssh channel created by on().  so you could just do:
```
on(agent, "PATH=\"#{agent['privatebindir']}:$PATH\" #{pxp_agent_dir(agent)}/pxp-agent --version")
```

if you need the PATH variable changed across on() calls, etc, you have to use something like:
```
host.add_env_var('PATH', '/opt/puppetlabs/bin:$PATH')
```